code/workhorse/screen_index_helper.R

```diff
@@ -0,0 +1,142 @@
+compute_covariance_h <- function(simulator,set1,set2){
+  #set2 can be pool or benchmark or the merge of these two
+  # set1= as.matrix(set1)
+  # set2 = as.matrix(set2)
+  
+  d = simulator$model[["numDim"]]
+  beta = simulator$model[["beta"]]
+  sigma = simulator$model[["sig2"]]
+  number = nrow(set1)
+  newcov=matrix(nrow=dim(set2)[1],ncol=number)
```

Comment:
do we have and oldcov also here? if not, maybe rename to `cov` or `cov_matrix`

Reply:
renamed.